mkdocs-base.yml

```diff
@@ -19,7 +19,7 @@ theme:
   favicon: _static/percona_favicon.ico
```

Comment:
Change to favicon: _static/percona-favicon.ico

Reply:
Done.